src/nrc/conf/includes/base.py

```diff
@@ -478,6 +479,13 @@
 # Celery
 CELERY_BROKER_URL = config("CELERY_BROKER_URL", "amqp://127.0.0.1:5672//")
 CELERY_RESULT_BACKEND = config("CELERY_RESULT_BACKEND", "redis://localhost:6379/1")
+CELERY_BEAT_SCHEDULE = {
```

Comment:
Please provide also tools for developers to run celery beat:
* bash file to run celery beat
* celery beat container in docker-compose.yml